PickNikRobotics/moveit_studio_ur_ws
# Copy example Objectives from Mimic Sim to Arm On Rail Sim

Resolves #313.

This PR ports the basic Objectives from `picknik_ur_mock_hw_config` to `arm_on_rail_sim`. The waypoints were also cleaned up. The `Constrained Pick and Place` Objective was updated to account for now actually repeatedly picking and placing a physical object.

### To test:
1) Bring up MoveIt Pro with this config.
2) Run the following Objectives and ensure that they succeed for atleast 1 cycle:
```
3 Waypoints Pick and Place
Constrained Pick and Place
Cycle Between Waypoints
Joint Diagnostic
```
